rooms.yml

```diff
@@ -481,6 +481,7 @@ meta.stackexchange.com:
       - 310369   # quartata
       - 350567   # iDebug (iBug on SO)
       - 377214   # Sonic the Anonymous Hedgehog (gparyani elsewhere)
+      - 163529   # Shree                                                                                                                                                                                                                                                                            
```

Comment:
It looks like you added lots of spaces after your name, is there any reason for that?